src/en/glossary.md

```diff
@@ -1,26 +1,23 @@
 # Glossary of terms
 
-**Bootstrap** - Initialize a Juju Environment so that Services may be deployed.
+**Bootstrap** - Initialize a Juju environment so that services may be deployed.
```

Comment:
you changed case on some words here, but not in the doc as a whole. I think there should be more consistency, but possibly capitalising them is not the answer. Maybe backticking them so they show up as highlighted in a monospaced font. My guess is they are to call out other terms which are defined on this page.